src/Component/OpenApi3/Tests/fixtures/github/expected/Model/ReactionRollup.php

```diff
@@ -111,41 +111,41 @@ public function setTotalCount(int $totalCount) : self
      *
      * @return int
      */
-    public function get1() : int
+    public function getN1() : int
```

Comment:
Getters & setters should still be `get1()` and `set1()`

Reply:
I face this problem:

- `getName()` can't be used because it the original name of the property; one the side effect is, for property like "+1" and "-1" (in the github dataset), it's cleaned as "1"; it generate two getters/setters with the same name
- using `getPhpName`, it contains the prefix.
- I put the prefix with "n" in the `Naming` class to avoid do it in many places...

What should I do?
- doesn't apply the cleaning in `getPropertyName`? it will not be stored in and returned by `getPhpName`
- or maybe add a new property on the `Property` class?

Reply:
I think it's worth having a new property on `Property` class, one for properties and one for methods :)

Reply:
Roger :ok_hand:

Reply:
I had to make `Naming::cleaning()` public.